src/Floating.svelte

```diff
@@ -56,6 +60,7 @@
   <div
     on:click={toggle}
     class="toggle"
+    class:highlight={expanded}
```

Comment:
Can we undo the changes in this file? I'll have another look when I redo the modals.

Reply:
4777bdf7